### PR TITLE
add content from 'fun...' and 'standard...'

### LINKED ISSUE #443: Required prerequisites for 'proving universality' are no longer in the ToC
The 'proving universality' section was written with the assumption that [fun with matrices](https://github.com/Qiskit/qiskit-textbook/blob/master/content/ch-gates/fun-matrices.ipynb) and [the standard gate set](https://github.com/Qiskit/qiskit-textbook/blob/master/content/ch-gates/proving-universality.ipynb) would be read first, but these are no longer in the table of contents.

Specifically, we are missing:

* The fact that `rx` gates can be expressed $R_x(\theta) = e^{i \frac{\theta}{2} X}$.
* The fact that any matrix can be written in terms of Paulis.

I'll have a go at scraping the required content out of the deprecated sections, and putting them into an extended 'proving universality'.

### PR DESCRIPTION
**Pull request by quantumjim**
Content from  'fun with matrices' and the 'standard gate set' that is required for 'proving universality' is added in to an extended version of 'proving universality' .

Fixes #443.

### CODE REVIEW

File: content/ch-gates/proving-universality.ipynb [hunk 1]
```diff
@@ -17,28 +17,302 @@
...
+    "\n",
+    "$$\n",
+    "CX_{j,k} ~(R_x(\\theta) \\otimes 1)~ CX_{j,k} = CX_{j,k} ~ e^{i \\frac{\\theta}{2} ~ (X\\otimes 1)}~ CX_{j,k} = e^{i \\frac{\\theta}{2} ~CX_{j,k} ~ (X\\otimes 1)~ CX_{j,k}} = e^{i \\frac{\\theta}{2} ~ X\\otimes X}\n",
+    "$$\n",
+    "\n",
+    "This transforms our simple, single-qubit rotation into a much more powerful two-qubit gate. This is not just equivalent to performing the same rotation independently on both qubits. Instead, it is a gate capable of generating and manipulating entangled states.\n",
+    "\n",
+    "We needn't stop there. We can use the same trick to extend the operation to any number of qubits. All that's needed is more conjugates by the CNOT to keep copying the $X$ over to new qubits.\n",
+    "\n",
+    "Furthermore, we can use single-qubit Cliffords to transform the Pauli on different qubits. For example, in our two-qubit example we could conjugate by $S$ on the qubit on the left to turn the $X$ there into a $Y$:\n",
```
Comment (HwajungKang) [inline content/ch-gates/proving-universality.ipynb:296]: we could conjugate by S on the qubit on the 'left' to turn the X there into a Y
   : but it looks like that the equation shows on the 'right' to turn..

File: content/ch-gates/proving-universality.ipynb [hunk 2]
```diff
@@ -17,28 +17,302 @@
...
+    "\n",
+    "$$\n",
+    "S ~e^{i \\frac{\\theta}{2} ~ X\\otimes X}~S^\\dagger = e^{i \\frac{\\theta}{2} ~ X\\otimes Y}.\n",
```
Comment (HwajungKang) [inline content/ch-gates/proving-universality.ipynb:299]: The equation might look clearer if it is written as
(I \otimes S) e^{i \frac{\theta}{2} X\otimes X} (I \otimes S^\dagger)  = e^{i \frac{\theta}{2}  X\otimes Y}

**Comment by quantumjim** [comment]
Thanks! done

**Comment by HwajungKang** [comment]
your welcome!

**Pull request merged**